### ISSUE TITLE
FCCS Fuel Type Id Meaning

### ISSUE THREAD

**Issue by Porkking** [pnwairfire/bluesky#258]
Hi Joel,
Does this BlueSKY package use the same fuel type id meaning as the online version (https://tools.airfire.org/playground/v3.5/emissionsinputs.php)?

If so, fccs map used in this package does not include all fuel types in the online version. I checked fccs2_fuelload.nc and it has 193 unique fuel types (including invalid fuel type: 0 and 900) while the online version has more. The online version even has some agricultural fuel types (eg: 1206 corn field). Also, do you know where to find the emission factors for these fuel types which are not included in current BlueSKY package?

Thanks,
Zongrun

**Comment by jdubowy** [comment]
You are correct: the FCCS ids represented in [fccsmap](https://github.com/pnwairfire/fccsmap) and thus bluesky are a subset of those listed in [bluesky playground](https://tools.airfire.org/playground/v3.5/emissionsinputs.php).   Playground has extra fuelbeds which appear in the dropdown but won't be assigned when looking up by map location (since that look-up uses bluesky/fccsmap).  When the emissions factors are looked up, in [eflookup](https://github.com/pnwairfire/eflookup), at least some of the extra FCCS ids (such as the agricultural ones) get mapped to the same covertypes as FCCS ids in fccsmap, and are assigned the same EFs.  e.g. 1290 through 1299 all get mapped to the same covertype as FCCS 79, 80, 81, and 83.

**Comment by jdubowy** [comment]
That's correct: the FCCS ids represented in [fccsmap](https://github.com/pnwairfire/fccsmap), and thus bluesky, are a subset of those listed in [bluesky playground](https://tools.airfire.org/playground/v3.5/emissionsinputs.php).   Playground has extra fuelbeds which appear in the dropdown but won't be assigned when looking up by map location (since that look-up uses bluesky/fccsmap).  When the emissions factors are looked up, in [eflookup](https://github.com/pnwairfire/eflookup), at least some of the extra FCCS ids (such as the agricultural ones) get mapped to the same covertypes as FCCS ids in fccsmap, and are assigned the same EFs.  e.g. 1290 through 1299 all get mapped to the same covertype as FCCS 79, 80, 81, and 83.
